include.js

```diff
@@ -6254,7 +6254,17 @@ let cards = [
     author: 'Robrigado',
     githubLink: 'https://github.com/Robrigado'
   }
```

Comment:
This introduces a syntax error. There is a comma missing.

On your computer, open the project in your terminal and run `npx prettier -w include.js` and it will show you the syntax errors.